### DISCUSSION TITLE
Sign out user everywhere?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [ory/kratos#1632] [Q&A] [answered]
Is there any way in Kratos to sign out a user everywhere? I.e. invalidate all sessions?

**Comment by danthegoodman1** [comment, marked as answer]
Found https://github.com/ory/kratos/issues/655

**Reply by vinckr** [reply]
Great that you found the issue right away! 
It is a pretty important feature and will hopefully be implemented soon. 
Contributions of any kind are welcome.
Let me know if you are interested or want to share your experience with Ory Kratos so far.
